### PR TITLE
Update to `grok 5.0`.

### PR DESCRIPTION
**Pull request by icemac**
This requires us to drop support for:
  - `grokcore.xmlrpc`
  - `grokcore.rest`
  - `grokcore.json`

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -1,6 +1,18 @@
 
 .. note:: The buildout versions file for all current versions can be found at https://zopefoundation.github.io/groktoolkit/
 
+4.0 (unreleased)
+----------------
+
+- Update to ``grok 5.0``. This requires us to drop support for:
```
Comment (janwijbrand) [inline docs/changes.rst:7]: we're going to 5.0 not 4.0?

**Review by janwijbrand — APPROVED** [review]
approved, with only a slight version 4.0 versus 5.0 confusion

File: docs/changes.rst (hunk 1 above)
Comment (icemac) [reply docs/changes.rst:7]: `grok 5.0` but `groktoolkit 4.0`.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**